features/facebookImport/src/model/analyses/off-facebook-events-analysis.js

```diff
@@ -1,17 +1,41 @@
+import React from "react";
+
 import RootAnalysis from "./root-analysis.js";
+import i18n from "../../i18n.js";
+import { groupOffFacebookEventsByType } from "./utils/off-facebook-events-utils.js";
+
+import OffFacebookMiniStory from "../../components/offFacebookMinistory/offFacebookMinistory.jsx";
 
 export default class OffFacebookEventsAnalysis extends RootAnalysis {
     get title() {
-        return "Off-Facebook Activity";
+        return i18n.t("offFacebookEventsMiniStory:title");
     }
 
     async analyze({ facebookAccount }) {
-        this._companiesCount = facebookAccount.offFacebookCompaniesCount;
-        this._eventsCount = facebookAccount.offFacebookEventsCount;
-        this.active = this._companiesCount > 0;
+        console.log(facebookAccount);
```

Comment:
CodeQL does no like this...

Reply:
Pretty bad also, getting a little unconcentrated 😅